R/poisson_reg-glmnet.R

```diff
@@ -82,6 +82,13 @@ multi_predict._fishnet <-
       }
     }
 
+    if (is.null(type)) {
+      type <- "numeric"
+    }
+    if (type == "numeric") {
+      dots$type <- "response"
+    }
```

Comment:
```suggestion
    if (type == "numeric") {
      # `glmnet:::predict.fishnet()` will receive `type = "response"`
      # via `parsnip:::predict_raw.model_fit` (#89)
      dots$type <- "response"
    }
```

Had to walk myself through the internals here to convince myself that something was actually happening to this list element.